sklearn/preprocessing/tests/test_label.py

```diff
@@ -502,7 +501,9 @@ def test_label_binarize_multilabel():
 
 def test_invalid_input_label_binarize():
     assert_raises(ValueError, label_binarize, [0, 2], classes=[0, 2],
-                  pos_label=0, neg_label=1)
+                  pos_label=0, neg_label=0)
+    assert_raises(ValueError, label_binarize, [0, 2], classes=[0, 2],
+                  neg_label=1, pos_label=-1)
```

Comment:
I find this harder to read because of switching neg_label and pos_label's order in the two applications here.